pytest.ini

```diff
@@ -1,4 +1,7 @@
 [pytest]
 addopts = --ds=hawc.main.settings.unittest --reuse-db --nomigrations
 python_files = tests.py test_*.py *_tests.py
-testpaths = tests
+testpaths =
```

Comment:
just curious, why is this change required? it looks like it skips integration tests- include?

Reply:
The integration tests are under frontend now, so they're still run. I had to make this change because after the restructure pytest would run things in a different order and cause problems. If we separate the paths like this pytest won't mix running the different kinds of tests we have, aka it will run all of the hawc tests, then integration tests, then client tests.

Reply:
ok, sounds good. thanks!